apps/api/src/hyp3_api/util.py

```diff
@@ -14,7 +14,7 @@ def get_granules(jobs: list[dict]) -> set[str]:
         granule
         for key in ['granules', 'reference', 'secondary']
         for job in jobs
-        for granule in job['job_parameters'].get(key, [])
+        for granule in job['job_parameters'].get(key, []) or []
```

Comment:
Slight simplification:
```suggestion
        for granule in job['job_parameters'].get(key) or []
```